primitives/linux-port/column.cpp

```diff
@@ -1421,7 +1421,8 @@ namespace primitives
 void PrimitiveProcessor::p_Col(NewColRequestHeader* in, NewColResultHeader* out,
                                unsigned outSize, unsigned* written)
 {
-    memcpy(out, in, sizeof(ISMPacketHeader) + sizeof(PrimitiveHeader));
```

Comment:
I'm surprised casting is necessary in all of these memcpy calls.  Do you know which version of gcc is complaining about this?

Reply:
All starting from gcc 7.

Reply:
pragma is another option how to disable warnings. But I prefer to have less pragmas disables the warnings throughout the code.